assets/js/components/GoogleChart/index.js

```diff
@@ -197,6 +212,114 @@ export default function GoogleChart( props ) {
 		}
 	}, [ onMouseOver, onMouseOut ] );
 
+	/**
+	 * Checks to see if the date is within the date range.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Date} date The date to check.
+	 * @return {boolean}  `true` if the date is within the current date
+	 *                    range, `false` otherwise.
+	 */
+	const isDateWithinRange = useCallback(
```

Comment:
Hmm - we don't usually provide a full JSDoc for functions defined within components, was this a case of autopilot?

Also - `addKeyDateLinesToChart()` does not use `useCallback()` (and nor should it), and this function also doesn't need to use it.

Reply:
No, it was ESLint complaining that I didn't document it when I used `/** */` style comments. I can nix it, but I've got into the habit of using `/** */` comments as VSCode will neatly use them when referencing the function/variables elsewhere.

But I'll remove it for consistency.